app/jobs/close_rejected_offender_sars_job.rb

```diff
@@ -0,0 +1,17 @@
+class CloseRejectedOffenderSarsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    offender_sars_to_close.each do |offender_sar|
+      offender_sar.update!(current_state: "closed", date_responded: Time.zone.today)
+      offender_sar.save!
+    end
+  end
+
+private
+
+  def offender_sars_to_close
+    _close_offender_sars = Case::SAR::Offender.where(current_state: "rejected")
```

Comment:
The assignment here is unnecessary as `_close_offender_sars` is not being used.

Reply:
removed